install.sh

```diff
@@ -191,6 +191,7 @@ git reset --hard origin/master
 yes | crew install buildessential
 yes | crew install less
 yes | crew install most
+yes | crew install $(curl -Ls https://github.com/skycocker/chromebrew/raw/master/tools/core_packages.txt)
```

Comment:
Why do we need this?  All of these packages should be installed already at this point.

Reply:
I've noticed that the installation doesn't install some of these packages. This just ensures that it does.

Reply:
How do you know core_packages.txt is correct?  I just used the output of `crew list installed` to create it after a fresh install.  Can you please substantiate the missing packages with some actual diffs?

Reply:
Here is your diff:

[diff](https://github.com/skycocker/chromebrew/files/2207388/diff.txt)

Reply:
Currently `readline` is not in core_packages.txt so maybe that needs updated instead. :)

Reply:
Here is the command I used to create the diff:
```
diff with-pr without-pr > diff
```